test/vision-model-checker.test.ts

```
@@ -0,0 +1,67 @@
+    process.env.NEXT_PUBLIC_VISION_MODELS = "custom-vision-model,another-vision-model";
+    
+    expect(isVisionModel("custom-vision-model")).toBe(true);
+    expect(isVisionModel("another-vision-model")).toBe(true);
+    expect(isVisionModel("unrelated-model")).toBe(false);
+  });
+
+  test("should handle empty or missing NEXT_PUBLIC_VISION_MODELS", () => {
+    process.env.NEXT_PUBLIC_VISION_MODELS = "";
+    expect(isVisionModel("unrelated-model")).toBe(false);
+
+    delete process.env.NEXT_PUBLIC_VISION_MODELS;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)